src/plugins/saved_objects_management/public/management_section/object_view/saved_object_view.tsx

```diff
@@ -91,7 +93,7 @@ export class SavedObjectEdition extends Component<
           canViewInApp={canView}
           type={type}
           onDeleteClick={() => this.delete()}
-          viewUrl={service.urlFor(id)}
+          viewUrl={http.basePath.prepend(object?.meta.inAppUrl?.path || '')}
```

Comment:
NIT: can we change `canViewInApp={canView && object?.meta.inAppUrl?.path}`

Reply:
thx, done in https://github.com/elastic/kibana/pull/89057/commits/cd503d51ae50c84bb01e042eed34b8770accc91f